src/wrapper/jnienv.rs

```diff
@@ -893,13 +908,13 @@ impl<'a> JNIEnv<'a> {
 
         let method_id: JMethodID = (class, ctor_sig).lookup(self)?;
 
-        self.new_object_by_id(class, method_id, ctor_args)
+        self.new_object_by_id_unchecked(class, method_id, ctor_args)
     }
 
     /// Create a new object using a constructor. Arguments aren't checked
     /// because
     /// of the `JMethodID` usage.
-    pub fn new_object_by_id<'c, T>(
+    pub fn new_object_by_id_unchecked<'c, T>(
```

Comment:
The consensus at the moment suggests we drop `_by_id` :-)

Reply:
Sounds pretty nice to me, I think this naming became a bit verbose. Let's wait for other opinions

Reply:
Let’s rename this ;-)